transport/src/main/java/io/camunda/zeebe/transport/stream/impl/LogicalId.java

```diff
@@ -17,4 +18,15 @@
  * @param metadata metadata of the stream
  * @param <M> type of metadata
  */
-record LogicalId<M>(UnsafeBuffer streamType, M metadata) {}
+record LogicalId<M>(UnsafeBuffer streamType, M metadata) {
+
+  @Override
+  public String toString() {
+    return "LogicalId{"
+        + "streamType="
+        + BufferUtil.bufferAsString(streamType)
```

Comment:
On the broker side it's a direct buffer because that's how we've serialized it to pass it around. On the client side we could probably just keep it a string, actually.

Reply:
I think it is ok to keep your changes. It is not a big problem.